pkg/provider/redis.go

```diff
@@ -60,12 +65,13 @@ func NewRedisClient(logger *zap.Logger, teamID string) (*RedisClient, error) {
 		client: rdb,
 		logger: logger,
 		teamID: teamID,
+		userID: userID,
```

Comment:
Do the `redis.NewClient` options support a namespace? Could utilize that for the teamId/userId as well